docs/docs/reference/command-line-interface.md

```diff
@@ -727,26 +727,8 @@ If the plugin you are trying to install declares that it does not support the ve
 
 :::info
 
-  <p>If you're using a custom Docker image, make sure `python3-venv` is installed:</p>
```

Comment:
Removed this cause I think it's irrelevant now that Meltano ships with `virtualenv` (https://github.com/meltano/meltano/pull/7198)